src/AvaTax.TaxModule.Data/Providers/AvaTaxRateProvider.cs

```diff
@@ -21,9 +21,9 @@ public class AvaTaxRateProvider : TaxProvider
         private readonly Func<IAvaTaxSettings, AvaTaxClient> _avaTaxClientFactory;
         private readonly AvaTaxSecureOptions _options;
 
-        public AvaTaxRateProvider() : base(nameof(AvaTaxRateProvider))
+        public AvaTaxRateProvider() 
```

Comment:
Need to call parameterless base constructor here

Reply:
Base constructor always calls first not dependent on writing ":base()" in code